biogeophys/EDSurfaceAlbedoMod.F90

```diff
@@ -363,10 +363,13 @@ subroutine PatchNormanRadiation (currentPatch, &
     cosz = max(0.001_r8, currentPatch%solar_zenith_angle ) !copied from previous radiation code...
     do ft = 1,numpft
        sb = (90._r8 - (acos(cosz)*180._r8/pi_const)) * (pi_const / 180._r8)
-       chil = xl(ft) !min(max(xl(ft), -0.4_r8), 0.6_r8 )
-       if ( abs(chil) <= 0.01_r8) then
-          chil  = 0.01_r8
-       end if
+       !chil should be between -0.4 and 0.6
+       !   Bonan (2019) doi:10.1017/9781107339217 pg. 238
+       chil = min(max(xl(ft), -0.4_r8), 0.6_r8 )
```

Comment:
I think since you've added the graceful failure, we don't really need to comparison here.

Reply:
okay I'll remove!